tests/functional/res/etl/d65ef9e6-3fd1-4026-9843-ff573c9e66c8/493a6adc-54b5-4388-ba11-c37686562127.zarr!expression_matrix!expression!.zarray

```diff
@@ -1,7 +1,7 @@
 {
     "chunks": [
-        10000,
-        10000
+        5,
+        20000000
```

Comment:
What does this do? It seems like this should make the tests fail?

Reply:
Oh yes this is weird. Updated to only restrict `n_cells`

Reply:
Oh yes this does produce incorrect output. Returned to normal chunk sizes